lms/djangoapps/courseware/views.py

```diff
@@ -726,6 +726,23 @@ def registered_for_course(course, user):
         return False
 
 
+def get_cosmetic_display_price(course, registration_price):
+    """
+    Return Course Price as a string preceded by correct currency, or 'Free'
+    """
+    currency_symbol = settings.PAID_COURSE_REGISTRATION_CURRENCY[1]
+
+    price = course.cosmetic_display_price
+    if registration_price > 0:
+        price = registration_price
```

Comment:
It's not *super* clear to me that's the behavior from this description on the Studio page:
https://cloud.githubusercontent.com/assets/3364609/5381467/5906944a-8056-11e4-9f07-8090f4910238.png

Can we reword that help line to convey the actual behavior better? Your explanation here makes sense, so I wonder how we can successfully convey that in the UX.